endor/src/index.js

```diff
@@ -23,7 +23,7 @@ import ClientService from './services/client.service';
 // eslint-disable-next-line import/no-unresolved
 import config from '../endorConfig.json';
 // eslint-disable-next-line import/no-unresolved
-import dbConfig from '../../dbConfig.json';
+import dbConfig from '../dbConfig.json';
```

Comment:
I overlooked this while fixing merge conflicts in a previous PR because the linter was disabled while tracking a file that's also gitignored. It's fixed now! :smiley: